cgroup.go

```diff
@@ -323,6 +323,9 @@ func (c *cgroup) Tasks(subsystem Name, recursive bool) ([]Task, error) {
 
 func (c *cgroup) processes(subsystem Name, recursive bool, pType procType) ([]Process, error) {
 	s := c.getSubsystem(subsystem)
+	if s == nil {
+		return nil, ErrControllerNotActive
+	}
 	sp, err := c.path(subsystem)
 	if err != nil {
 		return nil, err
```

Comment:
I was rethinking that error message. I think the following code might be more clear for the error

After Line 329,

if s == nil {
     return nil, errors.Errorf("no such cgroup %s in subsystem %s", sp, subsystem)
}

WDYT?

Reply:
Could we define a new Error with this?

Reply:
os.ErrNotExist for errors.Wrapf ?

Reply:
I think os.ErrNotExit is fine 😃

Reply:
I was thinking something like 
```
var ErrCgroupNotInSubsystem = errors.New("cgroups: cgroup not in subsystem")
```

Reply:
I was thinking to use templates to generate error by subsystem which make message more clear.

Reply:
> I was thinking something like 
> 
> ```
> 
> var ErrCgroupNotInSubsystem = errors.New("cgroups: cgroup not in subsystem")
> 
> ```

The error doesn't contain the which subsystem and which cgroup. That is My concern

Reply:
Yes, we should make it clear

Reply:
Could we use `fmt.Errorf("cgroups: %s doesn't exist in %s subsystem", sp, subsystem)` ?

Reply:
> Could we use `fmt.Errorf("cgroups: %s doesn't exist in %s subsystem", sp, subsystem)` ?

sgtm

Reply:
A new commit had been pushed.